src/core/domain/availability/tutorAvailabilitConts.ts

```diff
@@ -0,0 +1,14 @@
+import { CalendarOptions } from "../calendar";
```

Comment:
We do not have a domain in the ui lib, put them togother in the same folder and you can add a model folder, const, and needed folders inside the availability component folder